stdlib/experimental/polyline/polyline.flux

```diff
@@ -27,37 +27,42 @@ package polyline
 //
 // ## Examples
 //
-// ### Downsample the data using the epsilon value 1.5
+// ### Downsample data using the RDP algorithm
+//
+// When using `polyline.rdp()`, leave both `epsilon` and `retention` unspecified
+// to automatically calculate the maximum tolerance beyond which producing a
+// visually indistinguishable curve is not be possible.
```

Comment:
```suggestion
// visually indistinguishable curve would not be possible.
```

Reply:
perhaps:

to automatically calculate the maximum tolerance for producing a visually distinguishable curve.